config/locales/en.yml

```diff
@@ -1,153 +1,81 @@
-# encoding: UTF-8
-# Sample localization file for English. Add more files in this directory for other locales.
-# See https://github.com/svenfuchs/rails-i18n/tree/master/rails%2Flocale for starting points.
-
+---
 en:
-  back: Back
+  admin:
```

Comment:
`admin.save` is still in use in https://github.com/ODNZSL/nzsl-online/blob/staging/app/views/admin/settings/edit.html.haml#L14

Reply:
i think that's missing a called to t('admin.save'). I'm a bit surprised that works

Reply:
AHAH! went searching on apidock.com for submit_button - but it's really a helper method in this app.

i now wonder if i've missed other things translated via  `submit_button` I'll need to check more before we merge

Reply:
`submit_button` calls `t()` on it - see https://github.com/ODNZSL/nzsl-online/blob/staging/app/helpers/application_helper.rb#L14-L18

UI shows:

<img width="578" alt="screenshot 2019-01-01 09 52 54" src="https://user-images.githubusercontent.com/599867/50567703-edffa180-0dac-11e9-90c9-902b6d5eea16.png">